### ISSUE TITLE
Item: Berserkers Rage

### ISSUE THREAD

**Issue by warpdragon** [OpenAngelArena/oaa#1578]
# Berserker's Rage (Transformation Item)

- Behavior: Toggle
- Active Ability: On self()
  - minus_armor
  - mag_resist=0
  - status_resist=100%
  - attack_dmg x2)
- Passive Ability: Armlet stats
- Builds from: Armlet_2

**Comment by OTAJumperjay** [comment]
nobody is on this one yet .. yay i found something to learn coding on ... ez

**Comment by OTAJumperjay** [comment]
does the toggle have a cooldown?? like other transformation items or is it like regular armlet

**Comment by Haganeko** [comment]
Give it a cooldown. It can always be set to 0 later

**Comment by OTAJumperjay** [comment]
another thing should it set armor and magic res to 0 .. what i mean is if you get pipe for example would it still be 0 ?

**Comment by warpdragon** [comment]
in the spirit of the transformation yes i believe he should not benefit from Armor/MR from external sources or auras.

**Comment by Nortlen** [comment]
Should it increase bonus damage as well?

**Comment by mikibun** [comment]
Has this been finished?

**Comment by warpdragon** [comment]
No. Jay was hoping to learn to code in lua but this item was never finished. It was a very ambitious first project.

**Comment by khaostein** [comment]
Hello, is this still open? I want to try as a first project.

**Comment by mikibun** [comment]
@khaostein You're more than welcome to it as a first project

**Comment by khaostein** [comment]
Thank you. I will work on it
